{"name": "Принять заказ без id заказа", "status": "failed", "statusDetails": {"message": "AssertionError: assert (404 == 400)\n +  where 404 = <Response [404]>.status_code", "trace": "self = <tests.test_courier.TestCourier object at 0x000002CD7A7B6960>\n\n    @allure.title('Принять заказ без id заказа')\n    @allure.description('Попытка принять заказ без id заказа должна вернуть ошибку 400')\n    def test_accept_order_without_order_id_fail(self):\n        r = requests.post(TestCourierLinks.login_url, data=self.return_login_data())\n        courier_id = (r.json()['id'])\n        params = {'courierId': courier_id}\n        response = requests.put(OrdersLinks.accept_order,  params=params)\n>       assert response.status_code == 400 and response.json()['message'] == \"Недостаточно данных для поиска\"\nE       assert (404 == 400)\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_courier.py:130: AssertionError"}, "description": "Попытка принять заказ без id заказа должна вернуть ошибку 400", "start": 1701085496526, "stop": 1701085497283, "uuid": "c277187b-96bb-4b94-87f4-69a88ea67498", "historyId": "e44d2a9cdad30d24f1184b22588b37e9", "testCaseId": "e44d2a9cdad30d24f1184b22588b37e9", "fullName": "tests.test_courier.TestCourier#test_accept_order_without_order_id_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "Ya"}, {"name": "thread", "value": "27764-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}